### ISSUE TITLE
Not working with Sierra

### ISSUE THREAD

**Issue by SBenoit5** [jfro/QuickBoot#2]
I was happy to find your little app, but it's not working with Sierra 10.12.6. Has soon as I try to boot from Bootcamp, a warning appear : Failed to set boot volume, and the app crash. Would be nice if you could update it. Thanks.

**Comment by edenwaith** [comment]
From what I read at https://twocanoes.com/secureboot-imac-pro/, the "bless" command no longer works as of macOS 10.11, and the "systemsetup" command should be used, instead.  However, it looks like even systemsetup will not work on the new iMac Pros due to SecureBoot.

**Comment by SemoTech** [comment]
Is there any workaround for this?  Disabling SIP maybe?

**Comment by Madddcow** [comment]
Disabling SIP enables QuickBoot to work on Sierra and earlier, but not on High Sierra and later. Something must have changed from High Sierra.

**Comment by SemoTech** [comment]
Yes, exactly.  It's disappointing as I really miss QuickBoot...   Oh well, RIP!
